localstack/services/awslambda/lambda_utils.py

```diff
@@ -65,6 +65,10 @@
 DOCKER_MAIN_CONTAINER_IP = None
 LAMBDA_CONTAINER_NETWORK = None
 
+FUNCTION_NAME_REGEX = re.compile(
```

Comment:
I'd actually like to keep those as separated as possible for now since this provider + correlating code is a full own parallel replacement. So I'd like to avoid breaking the old provider when we move around some of our utils right now.

@dfangl wdyt?